lib/feature_management.rb

```diff
@@ -116,6 +116,10 @@ def self.idv_api_enabled?
     IdentityConfig.store.idv_api_enabled_steps.present?
   end
 
+  def self.idv_personal_key_confirmation_enabled?
+    IdentityConfig.store.idv_personal_key_confirmation_enabled
```

Comment:
since it's a boolean, should this call the `?` version of the method to be forthcoming?

Reply:
The convention (for better or worse) in our application.yml.default is to leave out the `?`s, but then the method we create in the FeatureManagement class (which is what we use in the rest of the code) does have the `?` to indicate a boolean.

Reply:
Related discussion: https://github.com/18F/identity-idp/pull/4887/files#r608867962